CTP/shell/src/com/navercorp/cubridqa/shell/deploy/DeployOneNode.java

```diff
@@ -38,18 +38,21 @@
 
 	Context context;
 	SSHConnect ssh;
+	String port, user, pwd;
 	String cubridPackageUrl;
 	boolean isNewBuildNumberSystem;
 
 	String envIdentify;
+	String currentEnvId;
 	Log log;
 
 	public DeployOneNode(Context context, String currEnvId, String host, Log log) throws Exception {
 		this.context = context;
+		this.currentEnvId =currEnvId;
 
-		String port = context.getProperty("env." + currEnvId + ".ssh.port");
-		String user = context.getProperty("env." + currEnvId + ".ssh.user");
-		String pwd = context.getProperty("env." + currEnvId + ".ssh.pwd");
+		port = context.getProperty("env." + currEnvId + ".ssh.port");
```

Comment:
As is:
port = context.getProperty("env." + currEnvId + ".ssh.port");
user = context.getProperty("env." + currEnvId + ".ssh.user");
pwd = context.getProperty("env." + currEnvId + ".ssh.pwd");

To be:
Use getInstanceProperty() method.
There are many such changes. We'd better replace all of them.